src/test/java/assertions/PersonTest.java

```diff
@@ -20,19 +25,27 @@ void getFullNameReturnsFirstnameSpaceLastname(){
     // --- getAge
 
     @Test
-    void getAgeReturns10YearsIfBornIn2009() throws Exception {
+    void getAgeReturns10YearsIfBornIn2011() throws Exception {
```

Comment:
stimmt der testmethodenname noch, wenn du den test in einem jahr wieder ausführst?